firmware_tools/ghidra/vxhunter_utility/symbol.py

```diff
@@ -1,34 +1,31 @@
 # coding=utf-8
-from ghidra.program.model.util import CodeUnitInsertionException
-from ghidra.program.model.data import (
-    CharDataType,
-    UnsignedIntegerDataType,
-    IntegerDataType,
-    UnsignedLongDataType,
-    ShortDataType,
-    PointerDataType,
-    VoidDataType,
-    ByteDataType,
-    ArrayDataType,
-    StructureDataType,
-    EnumDataType
-)
-from ghidra.program.model.symbol import RefType, SourceType
-from common import *
+
+import logging
 import string
+import struct
+import sys
 
+# Constants from common
+from common import can_demangle
+# Objects from common
+from common import demangler
+# Functions from common
+from common import is_address_in_current_program
+
+from ghidra.program.model.util import CodeUnitInsertionException
+from ghidra.program.model.data import (CharDataType, UnsignedIntegerDataType, IntegerDataType, UnsignedLongDataType, ShortDataType, PointerDataType, VoidDataType, ByteDataType, ArrayDataType, StructureDataType, EnumDataType)
```

Comment:
Please don't edit this kind of code, newline makes these types easier to add or delete.